<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="anilkumar06" name="learnJava.TestNGTest" tests="4" failures="1" timestamp="7 Aug 2019 04:54:59 GMT" time="0.023" errors="0">
  <testcase name="registrationTest" time="0.011" classname="learnJava.TestNGTest"/>
  <testcase name="hardAssertTest" time="0.004" classname="learnJava.TestNGTest"/>
  <testcase name="softAssertTest" time="0.006" classname="learnJava.TestNGTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Seems title not matched expected [Yahoo] but found [google],
	Seems image not matched expected [false] but found [true],
	Seems textbox not matched expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Seems title not matched expected [Yahoo] but found [google],
	Seems image not matched expected [false] but found [true],
	Seems textbox not matched expected [false] but found [true]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at learnJava.TestNGTest.softAssertTest(TestNGTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- softAssertTest -->
  <testcase name="signInTest" time="0.002" classname="learnJava.TestNGTest"/>
</testsuite> <!-- learnJava.TestNGTest -->
